.travis.yml

```diff
@@ -191,23 +185,43 @@ matrix:
           compiler: clang
 
         # Also regularly try the big-endian s390 architecture, in the
-        # process checking that installing dependencies with apt works.
+        # process checking that installing dependencies with apt works,
+        # and that astropy can properly use the system libraries.
+        # The latter is easy since the python3-astropy package, which is
+        # used to install the regular dependencies, also pulls those in.
+        # Hence, one can and should use ASTROPY_USE_SYSTEM_ALL=1.
         - name: big-endian s390x architecture with apt
           arch: s390x
           language: c
-          dist: bionic
-          stage: Cron tests
+          dist: focal
+          stage: Initial tests
```

Comment:
I think we need to move these back to cron?

Reply:
Yes, will do that the moment the general ideas here are approved!